src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -101,6 +103,33 @@ function mapStateToProps() {
     }
   );
 
+  const unusableExoticsSelector = createSelector(
```

Comment:
Was tempted to build this into the items selector so we only loop over all items once. This way separates concerns but loops twice and the logic is very similar.